e2etest/pkg/frr/bgp.go

```diff
@@ -63,3 +63,26 @@ func Routes(exec executor.Executor) (map[string]Route, map[string]Route, error)
 	}
 	return v4Routes, v6Routes, nil
 }
+
+// RawDump dumps all the low leven info as a single string.
+// To be used for debugging in order to print the status of the frr instance.
+func RawDump(exec executor.Executor) (string, error) {
+	res := ""
+	out, err := exec.Exec("vtysh", "-c", "show bgp neighbor")
+	if err != nil {
+		return "", errors.Wrapf(err, "Failed exec show bgp neighbor %s", res)
+	}
+	res = out
+	out, err = exec.Exec("vtysh", "-c", "show running-config")
+	if err != nil {
+		return "", errors.Wrapf(err, "Failed exec show bgp neighbor %s", res)
+	}
+	res = res + out
+	out, err = exec.Exec("cat", "/etc/frr/bgpd.conf")
+	if err != nil {
+		return "", errors.Wrapf(err, "Failed to cat bgpd.conf file %s", res)
+	}
```

Comment:
while you are there can add this to the e2e config template
log file /etc/frr/frr.log informational
log timestamp precision 3

and cat the file too to see any bgpd errors

Reply:
I will

Reply:
I am logging under /tmp , /etc/frr is mounted so I think we can't write there